Feature/polling atlas sensor

tudorsmt opened the pull request:
Polling sensor added. WIP

Commit:
added gitignore with python related ignores

Commit:
First commit of the ripe atlas polling sensor

The sensor uses a fixed Atlas measurement ID from which it interprets
the data. The sensor triggers on:
- Hops number changing between iterations
- Median RTT value changed between iterations
- Same iteration yields different values of the from field
- At least one of the test iterations has a different from value
than the expected one of the result.
- The host is detected to become partially reachable
- The host is detected to become partially unreachable

Commit:
Merge branch 'master' into feature/polling-atlas-sensor

tudorsmt commented on sensors/ripe_atlas_polling.py:
```diff
@@ -0,0 +1,208 @@
+    _measurement_delay_tolerance = 10
+
+    # tolerance, in ms, of rtt
+    _rtt_tolerance = -10 # TODO: roll back to 10 instead of -10
```
Fix this #TODO before merging.

Mierdin committed:
Adjust default setting for probe id in polling sensor

Signed-off-by: Matt Oswalt <matt@keepingitclassless.net>

Mierdin reviewed (approved):
There was a problem with the default value for the probe ID but I pushed ad0d9a5 to fix it. Otherwise seems to run, though I'm not seeing any trigger-instances - I'm assuming that's a threshold thing. Overall looks great!

tudorsmt commented:
Thanks for the change :)

Commit:
fixed typos in trigger parameter

Commit:
reverted the rtt tolerance to be 10 instead of -10

the -10 value was set for debugging purposes, so that the RTT trigger
always got triggered.

Commit:
Fixed TODOs in the descriptions of the triggers.

Commit:
Added rule that echoes the trigger values when RTT median drifts

Commit:
Fixed indents on hops_rtt_drift rule.

Commit:
host reachable/unreachable event has expected destination in payload

the expected destination is put into the payload of the triggered
trigger.

Commit:
Added todo for fixing naming of the partially reachable trigger

the partially reachable trigger represents the fully reachable state.

Commit:
Changed rule for hops_rtt_drift not to redirect to log file

the rule will simply echo the message.

Commit:
added example rules for the missing triggers

Rules that echo an error message were added. The addressed triggers:
- HopsNumberChanged
- FromFieldDifferentInAttempts
- FromFieldDifferentThanGeneral
- HostPartiallyUnreachable
- HostPartiallyReachable

Commit:
Fixed message for the median RTT value in rule
